prboom2/src/r_drawflush.inl

```diff
@@ -45,6 +43,76 @@
 //
 static void R_FLUSHWHOLE_FUNCNAME(void)
 {
+   // Scaled software fuzz algorithm
+#if (R_DRAWCOLUMN_PIPELINE & RDC_FUZZ)
+    if ((temp_x + startx) % fuzzcellsize)
+    {
+        return;
+    }
+
+    int yl = tempyl[temp_x - 1];
+    int yh = tempyh[temp_x - 1];
+
+    if (!yl)
+    {
+        yl = 1;
+    }
+
+    if (yh == viewheight - 1)
```

Comment:
To be honest, I'm not sure how "compositional" DrawColumn functions work in PrBoom+, but maybe we don't need the `if (yh == viewheight - 1)` check either.

Reply:
We dont need either of those checks. The values in temp_yl and temp_yh seems to always be valid.
Removed both conditions. I shouldve checked before